metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/InfluxDbReporter.java

```diff
@@ -246,15 +248,29 @@ public void report(
         }
     }
 
+    private void parse(String key, BiConsumer<String, Map<String, String>> report) {
+        String[] exploded = key.split("\\|");
+        Map<String, String> tags = new HashMap<>();
+        String name = exploded[0];
+        for (int i = 1; i < exploded.length; i++) {
+            String[] kv = exploded[i].split(":");
+            tags.put(kv[0], kv[1]);
```

Comment:
Hmm... that would need something like `Arrays.stream(exploded).skip(1).map(...)` - not sure how to do it exactly yet. But it feels a bit overkilling I guess?